File: 1233/1233.go
func removeSubfoldersFromTheFilesys(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := removeSubfoldersFromTheFilesys(root.Left)
	r := removeSubfoldersFromTheFilesys(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 1233/1233.cpp
class Solution {
public:
    int removeSubfoldersFromTheFilesys(TreeNode* root) {
        if(!root) return 0;
        return 1+max(removeSubfoldersFromTheFilesys(root->left),removeSubfoldersFromTheFilesys(root->right));
    }
};